src/main/kotlin/com/autonomousapps/DependencyAnalysisPlugin.kt

```diff
@@ -447,7 +447,8 @@ class DependencyAnalysisPlugin : Plugin<Project> {
       finalizedBy(failOrWarn)
 
       doLast {
-        logger.debug("Advice report (aggregated)  : ${adviceReport.get().projectReport.get().asFile.path}")
+        val reportPath = adviceReport.flatMap { it.projectReport.map { it.asFile.path } }
```

Comment:
In cases like this it makes sense to add `if (logger.isDebugEnabled())` or something like that to avoid computations when debug is disabled (which is the most common case)